### DISCUSSION TITLE
32-bit Linux

### DISCUSSION THREAD

**Discussion by GuzMan2019** [clangen/musikcube#478] [Ideas]
I have not found to download 32-bit version for Linux.

The Musikcube appearing to be very good! Why is it not in the list of "Application Manager" programs of Linux Mint?

**Comment by clangen** [comment]
Unfortunately we have not had packages accepted into any of the major Linux distributions; because of this, I have to build all the packages myself using my personal hardware, which takes quite a while for every release. A while back I decided to only provide pre-compiled binaries for 64-bit Linux to keep things manageable from my end.
